docs/commands/zed.md

```diff
@@ -854,11 +856,20 @@ pool `<existing>`, which may be referenced by its ID or its previous name.
 ```
 zed serve [options]
 ```
-The serve command implements Zed's server personality to service requests
+The `serve` command implements Zed's server personality to service requests
 from instances of Zed's client personality.
 It listens for Zed lake API requests on the interface and port
 specified by the `-l` option, executes the requests, and returns results.
 
+The verbosity of the server's log output can be varied with the `-log.level`
+option. Available levels ordered from most to least verbose are: `debug`, `info` (the
+default), `warn`, `error`, `dpanic`, `panic`, and `fatal`. Suggestions for use:
+
+* As its name implies, `debug` level is likely to only be required when actively debugging issues.
+
+* If the volume of logging output at the default `info` level seems too
+excessive for production use, `warn` level is recommended.
```

Comment:
These suggestions feel quite obvious. I wouldn't waste the reader's time with them.

Reply:
@nwt: I've shortened the suggestions to drop the `debug` mention (yeah, you're right that it's very obvious) but left the suggestion of `warn`. Quoting the user that spawned the issue:

> Is production, Zed Serve log to many info, I know that exist zed serve `-log.level=debug` to debug but, did exist a log level that show only warning and error? I searched on https://zed.brimdata.io/docs  I did not found details about log.level.
> After try and error, I found -log.level=warn look like what I need.

While it's possible they'd get what they need from the ordered list alone, I think it's worth making a specific recommendation to save them from hacking around with it more than necessary. Speaking just for myself, I've always had a terrible time remembering log levels and have to look them up every time, so I'd have wanted something like this if I was a user, too.